src/workers/atlas_worker.js

```diff
@@ -36,6 +37,10 @@ export default class AtlasWorker extends PeriodicWorker {
   }
 
   async periodicWork() {
+    if (!await this.checkIfEnoughFundsToPayForGas()) {
+      await this.addLog('Not enough funds to pay for gas');
```

Comment:
Oh, this is problematic. Basically we will spam MongoDB with this message every second. Totalling at 86400 entries a day until this is resolved. I think the check should be performed on every work loop. But the log should only happen on change enough > not-enough and not-enough > enough.